omod/src/main/java/org/openmrs/module/oauth2login/web/controller/CustomLogoutSuccessHandler.java

```diff
@@ -9,55 +9,26 @@
  */
 package org.openmrs.module.oauth2login.web.controller;
 
-import static org.openmrs.module.oauth2login.OAuth2LoginConstants.USER_PROP_ID_TOKEN;
-
 import java.io.IOException;
-import java.net.URLEncoder;
-import java.util.Properties;
 
 import javax.servlet.ServletException;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import org.apache.commons.lang3.StringUtils;
 import org.openmrs.api.context.Context;
-import org.openmrs.module.oauth2login.PropertyUtils;
+import org.openmrs.module.oauth2login.web.Utils;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.web.authentication.logout.LogoutSuccessHandler;
 import org.springframework.security.web.authentication.logout.SimpleUrlLogoutSuccessHandler;
-import org.springframework.util.MultiValueMap;
-import org.springframework.web.util.UriComponentsBuilder;
 
 public class CustomLogoutSuccessHandler extends SimpleUrlLogoutSuccessHandler implements LogoutSuccessHandler {
 	
 	@Override
 	public void onLogoutSuccess(HttpServletRequest request, HttpServletResponse response, Authentication authentication)
 	        throws IOException, ServletException {
-		Properties properties = PropertyUtils.getProperties(PropertyUtils.getOAuth2PropertiesPath());
```

Comment:
I just moved this code to a utility method in the Utils class so that we can reuse it from the filter for REST requests